source/boards/share-and-collaborate.rst

```diff
@@ -0,0 +1,101 @@
+
+To unlink the channel from the board, open the **Share** dialog, select the role dropdown next to the channel’s name, then select Unlink.
+
+.. note:: 
+  
+  A board can only be linked to one channel at a time. Linking another channel to the same board will automatically remove the link from the previous channel.
+
+Share a board
+--------------
+
+Boards can be shared internally with your team or published externally with limited accessibility.
+
+Share a board internally
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To share a board with team members internally, select **Share** in the top-right corner of the board, then select **Copy link** from the **Share** tab below. Paste the copied link in a channel or direct message to share the board with other team members. Only team members who have permissions to the board will be able to open the board from the shared link.
+
+Share a board publicly
+~~~~~~~~~~~~~~~~~~~~~~
+
+Sharing boards publicly is disabled by default. This means that the Publish tab is not available from the Share dialog. To enable public board sharing:
+
+1. Go to **Product menu > System Console > Plugins > Plugin Management > Mattermost Boards > Settings**.
+2. Set **Enable Publicly-Shared Boards** to **true**.
+3. Select **Save**.
+
+Once enabled, board admins can share a read-only link online with anyone: 
+
+1. Select **Share** in the top-right corner of the board.
+2. Toggle to the **Publish** tab.
+3. Switch to the **Publish to the web** option.
+4. Select **Copy link**.
+
+Paste the link anywhere you want to share the board. Anyone with the link will be able to view the board, but they won’t be able to edit the board.
+
+Select the **Regenerate Token** icon in the URL box if you want to invalidate all the previously shared links. Confirm the action to regenerate the token.
+
+Share cards on Channels
+-----------------------
+
+Cards can be linked and shared with team members directly on Mattermost Channels. When you share a link to a card within Channels, the card details are automatically displayed in a preview. This preview highlights what the card is about at a glance without having to navigate to it.
+
+To share a card, you'll need to copy the card link first:
+
+- Open a card and select the options menu **(...)** at the top right of the card, then select **Copy link**.
+- Alternatively, you can open the board view and hover your mouse over any card to access the options menu **(...)** for the card and select **Copy link** from there.
+
+After you've copied the link, paste it into any channel or direct message to share the card. A preview of the card will display within the channel with a link back to the card on Boards.
+
+Import and export a board
```

Comment:
@justinegeffen This section should be moved to the "Migrate boards" section. There's also updated content for that as well that should be added there